MAUI/DataGrid/Row-height-Customization.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+title: Row Height Customization in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Row Height Customization support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more.
+platform: MAUI
+control: SfDataGrid
+documentation: ug
+---
+
+# Row Height Customization in .NET MAUI DataGrid (SfDataGrid)
+
+The DataGrid provides an option to customize the header row height and the row height of all the grid rows or a particular row. To achieve this customization, see the following sections:
```

Comment:
The DataGrid provides an option to customize the header row height and the row height of all the grids or a particular row. To achieve this customization, see the following sections: